### DISCUSSION TITLE
[Enhancment] Change UI "actions"

### DISCUSSION THREAD

**Discussion by inson1** [hushenghao/AndroidEasterEggs#144] [Ideas]
I hate when Im scrolling and some card moves to left/right
I hate when I by mistake do long click and have to delete the created shortcut
There are 2 ways to create shortcut, moving left/right does only add one info (api version) and no new function
It would great to have more info like minimal needed android version for that easter egg and info about what that easter egg do

How to fix it?
Change this actions to open details after long click. 
The best would be new page.
Second would be details under the card.
In details would be Api version, Api version/Androud version needed for easter egg go work, written what the special thing can the easter egg do, last would be button for creating shortcut

PS If you dont like the idea please move it to discussion

**Comment by hushenghao** [comment]
Disabled longclick add shortcut 
commit: https://github.com/hushenghao/AndroidEasterEggs/pull/148/files#diff-00aef2584dda6bc2ce34ac0fc4aeb4377e193a4700646f9f81786c56e190a504
